ramalama.py

```diff
@@ -51,10 +52,17 @@ def main(args):
                            f"-v{home}:{home}",
                            "-v/tmp:/tmp",
                            f"-v{__file__}:/usr/bin/ramalama:ro",
-                           f"-v{wd}:{syspath}:ro",
-                           "-e", "RAMALAMA_HOST",
-                           "-p", f"{host}:{port}",
-                           "quay.io/ramalama/ramalama:latest", __file__] + args
+                           f"-v{wd}:{syspath}:ro"]
+            if os.path.exists("/dev/dri"):
+                conman_args += ["--device", "/dev/dri"]
+
+            if os.path.exists("/dev/kfd"):
+                conman_args += ["--device", "/dev/kfd"]
+
+            conman_args += ["-e", "RAMALAMA_HOST",
+                            "-p", f"{host}:{port}",
+                            "quay.io/ramalama/ramalama:latest", __file__]
```

Comment:
This __file__ should be "/usr/bin/ramalama" i believe.

Reply:
Or if you run from the git repo, it ends up being something like:

/Users/ecurtin/git/ramalama/./ramalama.py

it covers both the installed case and the development case when using `__file__`...